source/modules/assets/phones.rst

```diff
@@ -1,83 +1,239 @@
 Phones
 ======
 
+
+Phones can be registered manually or uploaded using the Android agent (not currently compatible with Apple terminals).
```

Comment:
I don't think any has used "terminals" to describe mobile devices in a while in English.
```suggestion
Phones can be registered manually or uploaded using the Android agent (no agent is currently compatible with iPadOS/iOS devices).
```

Also now that the Jamf plugin is officially maintained by Teclib' we can mention it here. Although Jamf is a different MDM software, we can import inventory data from it for Apple devices.